app/page.tsx

```
@@ -7,6 +7,7 @@ import { DashedHero } from "@/components/Hero/dashed-hero";
 import { MissionCard } from "@/components/mission-card";
 import { Newsletter } from "@/components/newsletter";
 import PowerBy from "@/components/PowerBy";
+import { Link } from "lucide-react";
```

ESLint

[error] 'Link' is defined but never used.

(@typescript-eslint/no-unused-vars)